Makefile

```
@@ -13,3 +13,7 @@ serve:
 .PHONY: build
 build:
 	yarn build
+
+.PHONY: lint
+build:
+	yarn lint
```

I like these Makefile wrappers in general, but the README seems to suggest calling the `yarn` commands directly. Could we go one way or the other? And if we keep the Makefile, make this little fix and adjust the README to use `make lint` (and probably also `make lint-fix`)? 

```suggestion
.PHONY: lint
line:
	yarn lint
```

---

Cool, removed this from the makefile